### PR TITLE
In the interface, No successfully, Enter

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/yubikey_setup.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Using YubiKey with the Journalist Interface
 ===========================================
 
 This is a quick and dirty guide to using YubiKey for two-factor
-authentication on the Journalist Interface.
+authentication in the Journalist Interface.
```
Comment (redshiftzero) [inline docs/yubikey_setup.rst]: "on the Journalist Interface" is better

Comment (comradekingu) [reply docs/yubikey_setup.rst]: I slightly favour in. However. "Add user page in the admin interface to add new users." is already used. Maybe change to "from" on both?

**Comment by dachary** [comment]
@comradekingu I can rebase this for you if you don't mind ?

**Comment by comradekingu** [comment]
@dachary Go ahead

**Comment by dachary** [comment]
Closed in favor of https://github.com/freedomofpress/securedrop/pull/2394 . Note that I did not pick the hunk related to the **in** vs **on** change. Feel free to submit a separate PR if you feel strongly about it and would like to discuss further.

**Pull request closed without merging**